client/my-sites/plans-features-main/index.jsx

```diff
@@ -341,7 +341,7 @@ class PlansFeaturesMain extends Component {
 	}
 }
 
-PlansFeaturesMain.PropTypes = {
+PlansFeaturesMain.propTypes = {
```

Comment:
There was a hidden bug here. React was ignoring all the specified `propTypes` because we specified them as `PropTypes`, not `propTypes`.